webpack/components/Errata/errataHelpers.js

```diff
@@ -0,0 +1,33 @@
+export const errataStatusContemplation = (errataStatus) => {
+  // from backend errata_status.rb:
+  // NEEDED_SECURITY_ERRATA = 3
+  // NEEDED_ERRATA = 2
+  // UNKNOWN = 1
+  // UP_TO_DATE = 0
+  const neededErrata = ([2, 3].includes(Number(errataStatus)));
+  const allUpToDate = (errataStatus === 0);
+  const otherErrataStatus = (!allUpToDate && !neededErrata);
+
+  return {
+    neededErrata,
+    allUpToDate,
+    otherErrataStatus,
+  };
+};
+
+export const friendlyErrataStatus = (errataStatus) => {
+  switch (errataStatus) {
+  case 0:
+    return 'All up to date';
+  case 1:
+    return 'Unknown';
+  // eslint-disable-next-line no-sequences
+  case 2:
```

Comment:
Could this be written like this for 2/3

`case (1 | 2) :`

Reply:
```js
const a = 2;
=> undefined
const b = 3;
=> undefined
console.log(a | b)
=> 3
```

Bitwise OR is just too confusing. It'd probably work with logical OR `(2 || 3)` but it feels safer to me to stick with the form in the MDN [examples](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch).